static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -239,6 +239,9 @@ export default function GroupSidebar({
         event={event}
         tagFormatter={TAGS_FORMATTER}
         project={project}
+        isStatisticalDetector={
+          group.issueType === IssueType.PERFORMANCE_DURATION_REGRESSION
+        }
```

Comment:
yeah if we start sending `template: "regression"` in the occurrence evidence we can use it instead of group type. not to self